smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -253,6 +258,13 @@ public async Task Configure(DeviceProvisioningMethod method, Option<string> agen
                 doc.ReplaceOrAdd("listen.management_uri", uris.ListenManagement);
                 doc.ReplaceOrAdd("listen.workload_uri", uris.ListenWorkload);
             }
+            else if (hostname.Contains("-EFLOW-"))

```

Comment:
So we require that the agent's hostname contains "-EFLOW-"? That seems sketchy...is there a better way to differentiate eflow from others when building config.yaml?

Reply:
Added section that checks for a specific string in /etc/os-subrelease to determine if the system is EFLOW.